src/index.js

```diff
@@ -5,5 +5,13 @@ import registerServiceWorker from 'registerServiceWorker';
 // @TODO: Remove this after creating the `Button` component
 import 'theme/index.scss';
 
+var WebFont = require('webfontloader');
+
+WebFont.load({
+  google: {
+    families: ['Lato', 'Fjalla One'],
+  },
+});
```

Comment:
I think we can use `sessionStorage` option here like:

https://github.com/robinrendle/font-loading/tree/master/examples/web-font-loader#3-add-settings-to-the-webfontconfig-object